tests/integration/api/test_credential.py

```diff
@@ -50,12 +52,31 @@ def test_create_credential(client: APIClient):
         "username": "me",
         "credential_type": CredentialType.REGISTRY,
         "id": id_,
+        "identifier": None,
     }
     obj = models.Credential.objects.filter(pk=id_).first()
     assert obj.username == "me"
     assert obj.secret == "mypassword"
 
 
+@pytest.mark.django_db
```

Comment:
@hsong-rh Can we please add a test for Vault without an identifier since the identifier can be empty.
<img width="1630" alt="Screenshot 2024-01-23 at 4 55 02 PM" src="https://github.com/ansible/eda-server/assets/6452699/22471fbe-3dd4-4e1b-94b6-7e9cc1d2c319">

Reply:
@hsong-rh Can the credential be updated after its created, initially its created without the identifier but then the user decides to add an identifier during an update

Reply:
@mkanoor added